doc/docs/master/getting_started/install-pachctl-completion.md

```diff
@@ -0,0 +1,55 @@
+# Install `pachctl` Autocompletion
+
+Pachyderm autocompletion allows you to automatically finish
+partially typed commands by pressing `TAB`. Autocompletion needs
+to be installed separately when `pachctl` is already
+available on your client machine.
+
+Pachyderm autocompletion is supported for both `bash` and `zsh`.
+You must have either of them preinstalled
+before installing Pachyderm autocompletion.
+
+!!! tip
+    Type `pachctl completion --help` to display help information about
+    the command.
+
+To install `pachctl` autocompletion, perform the following steps:
+
+1. Verify that `bash` or `zsh` completion is installed on your machine.
+   For example, if you have installed bash completion by using Homebrew,
+   type:
+
+   ```bash
+   brew info bash-completion
+   ```
+
+   This command returns information about the directory in which
+   `bash-completion` is installed.
```

Comment:
the `bash-completion` package and bash completion scripts are installed in separate locations. `brew info bash-completion` will return info about both paths.